src/service/SecureKeychain.ts

```diff
@@ -0,0 +1,107 @@
+import {Platform} from 'react-native';
+import * as Keychain from 'react-native-keychain';
+import {encrypt, decrypt} from 'src/service/Encryptor';
+import {mmkvStorage} from 'src/service/MMKVStorage';
+
+const SALT = 'secret_phrase';
```

Comment:
We could probably get this from a build variable, but this is used just as a phrase to "encode" the password to add an extra layer of security. They keychain is already encrypting the data.

Reply:
Do you mean using env vars on build to inject the salt? I think that would also accessible as they explain [in react native docs](https://reactnative.dev/docs/security#storing-sensitive-info).

Reply:
Yeah I know it will be accessible, this case is something we have to live with (unless we ask the user for a password to encrypt the password 😆 )